<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="deixei_sharp_testing_for_pytest" errors="1" failures="0" skipped="0" tests="1" time="0.053" timestamp="2024-05-05T12:16:47.987350" hostname="dx2023"><testcase classname="test_my_test_plan2.test_my_test_suite_3" name="test_my_test_case_5" time="0.002"><error message="failed on setup with &quot;file /home/marcio/repos/deixei/sharp-testing/tests/test_my_test_plan2/test_my_test_suite_3.py, line 6&#10;  @pytest.mark.test_id(29)&#10;  def test_my_test_case_5(ado_config, test_run_id, test_result_id, ado_url, ado_pat, ado_project, verbose, azure_tenant, azure_client_id, azure_secret):&#10;E       fixture 'verbose' not found&#10;&gt;       available fixtures: ado_config, ado_pat, ado_project, ado_url, azure_client_id, azure_secret, azure_tenant, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, print_verbose, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, test_result_id, test_run_id, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory&#10;&gt;       use 'pytest --fixtures [testpath]' for help on them.&#10;&#10;/home/marcio/repos/deixei/sharp-testing/tests/test_my_test_plan2/test_my_test_suite_3.py:6&quot;">file /home/marcio/repos/deixei/sharp-testing/tests/test_my_test_plan2/test_my_test_suite_3.py, line 6
  @pytest.mark.test_id(29)
  def test_my_test_case_5(ado_config, test_run_id, test_result_id, ado_url, ado_pat, ado_project, verbose, azure_tenant, azure_client_id, azure_secret):
E       fixture 'verbose' not found
&gt;       available fixtures: ado_config, ado_pat, ado_project, ado_url, azure_client_id, azure_secret, azure_tenant, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, print_verbose, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, test_result_id, test_run_id, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
&gt;       use 'pytest --fixtures [testpath]' for help on them.

/home/marcio/repos/deixei/sharp-testing/tests/test_my_test_plan2/test_my_test_suite_3.py:6</error></testcase></testsuite></testsuites>